1_crosswalk_fetch/src/get_ned_points.R

```diff
@@ -0,0 +1,26 @@
+
+#' get point elevation data from the `elevatr` package
+#'
+#' @param point_rds spatial data file for points
+#' @param zoom zoom level, input into `elevatr::get_aws_points` function
+#' @param dummy character that isn't used but if changed, will cause rebuilds
+#'
+#' @details
+#' needed to install CRAN binary of sf and downgrade PROJ/GDAL to get past error:
+#' Error in sf::gdal_utils(util = "warp", source = files,
+#' destination = destfile,  : gdal_utils warp: an error occured
+#' Captured later here: https://github.com/jhollist/elevatr/issues/62
+get_ned_points <- function(point_rds, zoom, dummy){
+
+  point_rds %>% readRDS() %>%
+    elevatr::get_aws_points(z = zoom, verbose = FALSE) %>%
+    suppressMessages() %>% {.[[1]]} %>%
```

Comment:
yes, the `{}` allows you to widen the scope of the thing you are piping the result of `suppressMessages` to. So in this case, I want the scope to include the `[[1]]`, which wouldn't be applied if I'd not used the brackets here. 

It can also be used to snag the piped value more than once, like this:

```r
list(x=1, y=3) %>% {plot(.$x, .$y)} # plots
```
while this fails:
```r
list(x=1, y=3) %>% plot(.$x, .$y)
Error in xy.coords(x, y, xlabel, ylabel, log) : 
  'x' and 'y' lengths differ
```